config/router.config.js

```diff
@@ -80,6 +80,11 @@ module.exports = [
         name: 'share',
         component: './SessionPlaceholder',
       },
+      {
+        path: '/system-comparisions',
```

Comment:
Typo, comparisions -> comparisons, needs fixing in multiple places

Reply:
@mnmehta My bad, will fix that